### PR TITLE
Add bmbegin and end, and sentence form "the battery…

### PR DESCRIPTION
**Pull request by NoSloppy**
… level is"

The prompt battlevl.wav is built into the Voicepaks.

### CODE REVIEW

File: sound/sound_library.h [hunk 1]
```diff
@@ -143,6 +143,9 @@ class SoundLibrary {
   void SayAuto() { Play("mauto.wav"); }
   void SayBaseColor() { Play("mbase.wav"); }
   void SayBatteryLevel() { Play("mbatt.wav"); }
+  void SayTheBatteryLevelIs() { Play("battlevl.wav"); }
+  void SayBattleModeBegin() { Play("bmbegin.wav"); }
```
[the remark this answers is not in the document]
Comment (NoSloppy) [reply sound/sound_library.h:147]: Actually, it just says "battle mode".
So SayBattleMode?

[the remark this answers is not in the document]
Comment (NoSloppy) [reply sound/sound_library.h:147]: And for that matter, if it's the literal phrase, the other one should change to SayEndBattleMode

**Comment by fdarosa2663** [comment]
I may have mixed Brian up while we were creating the sounds.  the bmbegin and bmend and vmbegin and vmend got into the list I gave him, but we'll want to remove those as they have a different use from OS5.

**Comment by NoSloppy** [comment]
Trying again.

**Pull request closed without merging**